processing/Dockerfile

```diff
@@ -14,23 +14,15 @@ ENV PYTHONUNBUFFERED 1
 FROM base as build
 COPY alembic alembic
 COPY src src
-CMD alembic upgrade head && uvicorn src.main:app --host 0.0.0.0 --port 8080
+CMD alembic upgrade head && uvicorn src.main:app --host 0.0.0.0 --port 8080 --reload
```

Comment:
May you clarify usage of `reload` here?

Reply:
I used it during development so the webapp could reload after changes in code.
I've deleted it here